nix/internal/darwin-terminal-init.sh

```diff
@@ -13,16 +13,18 @@ color_reset=$'\e[0m'
 
 cat <<EOF
 
-  Hello!
+  Welcome!
 
-  This terminal has been set up to allow easy running of ‘${color_bold}blockfrost-platform${color_reset}’.
+  This terminal is set up for easy execution of ‘${color_bold}blockfrost-platform${color_reset}’.
 
-  Try typing in one of:
+  Run one of the following commands:
 
-    ${color_bold}blockfrost-platform --help${color_reset}
-    ${color_bold}blockfrost-platform --init${color_reset}
+   For setup: ${color_bold}blockfrost-platform --help${color_reset}
+   For help: ${color_bold}blockfrost-platform --init${color_reset}
```

Comment:
Would it make sense to keep the commands vertically aligned? I.e. add one more space after `help:`.